gusto/time_discretisation.py

```diff
@@ -323,13 +323,37 @@ def setup(self, equation, apply_bcs=True, *active_labels):
 
         super().setup(equation, apply_bcs, *active_labels)
 
+        # Get continuity form transport term
+        for t in self.residual:
```

Comment:
As we discussed offline: it might be best to move this code to break up the continuity form out of time discretistations to a new routine (we suggested that it would be in `common_forms.py`). Then a `TimeDiscretisation` can be agnostic to the terms that it works on, and this code than then be reused for other things.